src/components/button/button.component.js

```diff
@@ -50,6 +57,11 @@ const Button = (props) => {
     buttonType: buttonType || as
   };
 
+  if (fullWidth && size !== 'large') {
```

Comment:
It should also error if the buttonType is not `dashed`

Reply:
fixed